vali_config.py

```diff
@@ -83,9 +83,27 @@ def pair_map():
         return {pair.trade_pair_id: pair for pair in TradePair}
 
     @staticmethod
-    def get_trade_pair(trade_pair_id: str):
+    def from_string(trade_pair_id: str):
```

Comment:
can we rename this from_trade_pair_id to make it clearer

Reply:
done